### PR TITLE
CTECH-2605 Added notebook for T11002 Working with Quotes

### PR DESCRIPTION
**Pull request by MikeMcGarry**
# Pull Request Checklist

- [ ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb [hunk 1]
```diff
@@ -0,0 +1,1262 @@
+{
```
Comment (rizwansaeed) [inline university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: <p>do we need to fix up the formatting as per the error message?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (rizwansaeed) [inline university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.        max_age="P20Y1M1DT1H1M1.1S",
</pre><p>wow didnt know you could do this!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (rizwansaeed) [inline university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: <p>is the failed delete expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (MikeMcGarry) [reply university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: Yes

Comment (MikeMcGarry) [reply university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: Me neither, pretty cool!

Comment (MikeMcGarry) [reply university/T11002-WorkingWithQuotes/T11002-WorkingWithQuotes.ipynb:1]: No this is intentional to demonstrate a partial success.

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**